diff --git a/src/vs/platform/window/common/window.ts b/src/vs/platform/window/common/window.ts
index d547c37bf50..9b2b3f504b2 100644
--- a/src/vs/platform/window/common/window.ts
+++ b/src/vs/platform/window/common/window.ts
@@ -222,7 +222,7 @@ export function getTitleBarStyle(configurationService: IConfigurationService): T
 		}
 	}
 
-	return isLinux && product.quality === 'stable' ? TitlebarStyle.NATIVE : TitlebarStyle.CUSTOM; // default to custom on all OS except Linux stable (for now)
+	return isLinux && product.quality === 'stable' ? TitlebarStyle.CUSTOM : TitlebarStyle.CUSTOM;
 }
 
 export const DEFAULT_CUSTOM_TITLEBAR_HEIGHT = 35; // includes space for command center
